pkger/testdata/dashboard_mosaic.yml

```diff
@@ -18,13 +18,19 @@ spec:
         - xTickStart
         - xTickStep
       xCol: _time
+      hoverDimension: "y"
       xTotalTicks: 15
       xTickStart: 0
       xTickStep: 1000
       yCol: _value
       width:  6
       height: 3
-      ySeriesColumns: ["_value", "foo"]
+      yLabelColumnSeparator: ","
+      yLabelColumns:
+        - foo
+      ySeriesColumns:
+        - _value
+        - foo
```

Comment:
If you leave the test and this file unchanged, does the existing test logic still pass?

Reply:
Yes.